packages/core/examples/dialogExample.tsx

```diff
@@ -19,7 +19,7 @@ export class DialogExample extends OverlayExample {
                     className={this.props.getTheme()}
                     iconName="inbox"
                     onClose={this.handleClose}
-                    title="Dialog Header"
+                    title="Dialog header"
```

Comment:
👎  these text changes really don't belong here. would be great if you could pull them out to a separate PR so this PR contains literally only what is in the title.